cmd/protoc-gen-go-tableau-loader/hub.go

```diff
@@ -137,12 +138,14 @@ func BoolToInt(ok bool) int {
 
 // Hub is the messager manager.
 type Hub struct {
-	messagerMap MessagerMap
+	messagerMap    MessagerMap
+	lastLoadedTime time.Time
 }
 
 func NewHub() *Hub {
 	return &Hub{
-		messagerMap: MessagerMap{},
+		messagerMap:    MessagerMap{},
+		lastLoadedTime: time.Now(),
```

Comment:
default value should be `time.Unix(0, 0)`

Reply:
done